README.md

```diff
@@ -34,7 +34,23 @@ Supports popular public checkpoints such as:
 
 This implementation extends on [Lit-LLaMA](https://github.com/lightning-AI/lit-llama) and [nanoGPT](https://github.com/karpathy/nanoGPT), and it's **powered by [Lightning Fabric](https://lightning.ai/docs/fabric/stable/) ⚡**.
 
-## Design principles
+
+
+---
+
+**NeurIPS 2023 Large Language Model Efficiency Challenge: 1 LLM + 1 GPU + 1 Day**
+
+Lit-GPT is the official starter kit of the [NeurIPS 2023 LLM efficiency challenge](https://llm-efficiency-challenge.github.io), which is competition centered around training a foundational (base) LLM for 24h on a single A100 or 4090.
```

Comment:
```suggestion
Lit-GPT is the official starter kit of the [NeurIPS 2023 LLM efficiency challenge](https://llm-efficiency-challenge.github.io), which is a competition centered around training a foundational (base) LLM for 24h on a single A100 or 4090.
```

Reply:
Maybe it's just me, but the phrase:
> training a foundational (base) LLM for 24h on a single A100 or 4090

for me means that the foundational model will be trained from scratch.
I think we already have the convention of what is training, transfer learning, fine-tuning, ...

Maybe something like:
> which is a competition centered around fine-tuning a foundational (base) LLM to specific tasks for 24h on a single A100 or 4090.

Or something similar. Your English is better then mine.